tools/monorepo-utils/tsconfig.json

```diff
@@ -0,0 +1,10 @@
+{
+	"compilerOptions": {
+		"rootDir": "src",
+		"outDir": "dist",
+		"target": "es6",
+		"module": "commonjs",
+		"esModuleInterop": true,
+		"moduleResolution": "node"
```

Comment:
Take a look at the other `tsconfig.json` files for packages and note the `"typeRoots"` settings there. You'll want to do that similarly here for package isolation.